config/settings.yml

```diff
@@ -1,6 +1,6 @@
 hostname: 127.0.0.1:3000 # possible fallback for unsafe request.host
 
-vsp_environment: 
```

Comment:
This looks like some kind of typo/editor change?

Reply:
I'm not entirely sure. I will check up on it!

Reply:
Looks like my editor just stripped that additional trailing white space.